### PR TITLE
Task/3657 contact section new design

### PR DESCRIPTION
**Pull request by tiwariav**

### CODE REVIEW

**Comment by verysanghee** [comment]
background color needs to be brightend

File: datacenterlight/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -82,6 +82,21 @@ msgstr "Bitte gib eine gültige E-Mailadresse ein."
...
 msgstr "Weiter"
 
+msgid "Thank you for contacting us."
+msgstr "Nachricht gesendet."
+
+msgid "Your message was successfully sent to our team."
+msgstr "Vielen Dank für Deine Nachricht."
+
+msgid "Get in touch with us!"
+msgstr "Kontaktiere uns"
```
Comment (verysanghee) [inline datacenterlight/locale/de/LC_MESSAGES/django.po]: "Sende uns eine Nachricht."

**Comment by verysanghee** [comment]
- Email, Name, Message labels letter-spacing: 0.6
- When clicked there's a dot(.) appearing on the btn/ letters move to the left.
- Change facebook logo into round one (will send you via chat, github doesn't support .svg upload)

**Comment by verysanghee** [comment]
Really cool transition for success message btw!!thumbs up!

File: datacenterlight/templates/datacenterlight/index.html [hunk 2]
```diff
@@ -154,9 +154,26 @@ <h2 class="section-heading">{% trans "Simple and affordable: Try our virtual mac
...
+                                <p>In der Au 7, Schwanden 8762</p>
+                                <p>{% trans "Switzerland " %}</p>
+                            </div>
+                        </div>
+                        <div class="social">
+                            <a target="_blank" class="" href="https://twitter.com/datacenterlight"><i class="fa fa-twitter fa-fw"></i></a>
+                            <a target="_blank"  class=""  href="https://github.com/ungleich"><i class="fa fa-github fa-fw"></i></a>
+                            <a target="_blank"  class=""  href="https://www.facebook.com/ungleich.ch/"><i class="fa fa-facebook fa-fw"></i></a>
+                        </div>
+                        <!-- <div class="card">
```
Comment (pcoder) [inline datacenterlight/templates/datacenterlight/index.html]: I guess it would be clean to remove this?

**Pull request closed without merging**